src/App/Components/Common/NavProfile/navLink.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+
+
+export class navLink extends Component {
+  render() {
+    return (
+
+      <div className="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul
+          className="navbar-nav mr-auto"
+          style={{ marginLeft: '10%' }}
+        >
+          <li className="nav-item active">
+            <a className="nav-link" href="/">
+               HOME
```

Comment:
@Ramadhan0 why are these hard-coded?

Reply:
I wanted to add elements in navbar but @nkpremices is the one who assigned the task @MemunaHaruna

Reply:
I going to remove this file